### DISCUSSION TITLE
New model support: Support for inference with TESTR (Mutually exclusive with the next task).  新模型支持：支持 TESTR 的推理

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#290] [MMOCR]
# New model support: Support for inference with TESTR (Mutually exclusive with the next task).  
 新模型支持：支持 TESTR 的推理
 

*Referring to the official implementation of TESTR, implement the inference code in the projects/ directory. The results should align with the official accuracy. The weights can be directly obtained by converting them from the official script.*
*References:*
- *Paper:*  [https://openaccess.thecvf.com/content/CVPR2022/html/Zhang_Text_Spotting_Transformers_CVPR_2022_paper.html](https://openaccess.thecvf.com/content/CVPR2022/html/Zhang_Text_Spotting_Transformers_CVPR_2022_paper.html) 
- *Official implementation:* [https://github.com/mlpc-ucsd/TESTR](https://github.com/mlpc-ucsd/TESTR) 

（与 [https://github.com/open-mmlab/OpenMMLabCamp/discussions/290](https://github.com/open-mmlab/OpenMMLabCamp/discussions/290)  互斥）参考 TESTR 的官方实现，在 projects/ 文件夹中实现推理部分代码，结果需要对齐官方精度。权重可直接通过脚本从官方转换得到。
参考：
- 论文: [https://openaccess.thecvf.com/content/CVPR2022/html/Zhang_Text_Spotting_Transformers_CVPR_2022_paper.html](https://openaccess.thecvf.com/content/CVPR2022/html/Zhang_Text_Spotting_Transformers_CVPR_2022_paper.html) 
- 官方实现：[https://github.com/mlpc-ucsd/TESTR](https://github.com/mlpc-ucsd/TESTR) 

## Task Information

**Repository**：MMOCR

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：7 days

## Task Status

**Claimant's Github ID**：xiangyw99

**Claim Time**：2023年10月08日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by xiangyw99** [comment]
Claim the mission
